src/gtk/window.cpp

```diff
@@ -3302,6 +3303,41 @@ touch_callback(GtkWidget* WXUNUSED(widget), GdkEventTouch* gdk_event, wxWindowGT
     if ( !data )
         return;
 
+    if ( data->m_rawTouchEvents)
+    {
+        wxEventType type;
+
+        switch(gdk_event->type)
+        {
+        case GDK_TOUCH_BEGIN:
+            type = wxEVT_TOUCH_BEGIN;
+            break;
+
+        case GDK_TOUCH_UPDATE:
+            type = wxEVT_TOUCH_MOVE;
+            break;
+
+        case GDK_TOUCH_END:
+            type = wxEVT_TOUCH_END;
+            break;
+
+        case GDK_TOUCH_CANCEL:
+            type = wxEVT_TOUCH_CANCEL;
+            break;
+
+	default:
+	    type = wxEVT_NULL;
+        }
+	if (type != wxEVT_NULL)
+	{
+            wxMultiTouchEvent event(type);
+	    event.SetPosition(wxPoint(gdk_event->x, gdk_event->y));
```

Comment:
And here as well. I won't repeat this comment any more, but please check that you don't have any hard TABs in your changes and replace them with spaces if you do. TIA!